modules/ROOT/pages/clauses/match.adoc

```diff
@@ -262,151 +228,262 @@ RETURN a, b
+[source, cypher]
 ----
-MATCH (wallstreet:Movie {title: 'Wall Street'})<-[:ACTED_IN]-(actor)
-RETURN actor.name
+MATCH (:Movie {title: 'Wall Street'})<-[:ACTED_IN]-(actor:Person)
+RETURN actor.name AS actor
 ----
 
-Returns all actors who `ACTED_IN` the movie `Wall Street`.
-
 .Result
-[role="queryresult",options="header,footer",cols="1*<m"]
+[source, role="queryresult",options="header,footer",cols="1*<m"]
 |===
-| actor.name
+| actor
 | "Michael Douglas"
 | "Martin Sheen"
 | "Charlie Sheen"
 |Rows: 3
 |===
 
-Read more about xref:patterns/fixed-length-patterns.adoc#relationship-patterns[relationship type expressions].
-
-[[match-on-multiple-rel-types]]
-=== Match on multiple relationship types
+[[match-on-multiple-relationship-types]]
+=== Filter on multiple relationship types
 
-It is possible to match on multiple relationship types by using the pipe symbol (`|`).
-For example:
+It is possible to match for a pattern containing one out of several relationship types using the `OR` symbol, `|`.
 
-.Query
-[source, cypher, indent=0]
+.Relationship pattern including either `ACTED_IN` or `DIRECTED` relationship types
+[source, cypher]
 ----
-MATCH (wallstreet {title: 'Wall Street'})<-[:ACTED_IN|DIRECTED]-(person)
-RETURN person.name
+MATCH (:Movie {title: 'Wall Street'})<-[:ACTED_IN|DIRECTED]-(person:Person)
+RETURN person.name AS person
 ----
 
-Returns nodes with an `ACTED_IN` or `DIRECTED` relationship to the movie `Wall Street`.
-
 .Result
 [role="queryresult",options="header,footer",cols="1*<m"]
 |===
-| person.name
+| person
 | "Oliver Stone"
 | "Michael Douglas"
 | "Martin Sheen"
 | "Charlie Sheen"
 |Rows: 4
 |===
 
+As relationships have exactly one type each, `()-[:A&B]->()` will never match a relationship.
 
-[[match-on-rel-type-use-variable]]
-=== Match on relationship type and use a variable
+[[multiple-relationships]]
+=== Find multiple relationships
 
-Variables and specific relationship types can be included in the same pattern.
-For example:
+A graph pattern can contain several relationship patterns.
 
-.Query
-[source, cypher, indent=0]
+.Graph pattern including several relationship patterns
+[source, cypher]
 ----
-MATCH (wallstreet {title: 'Wall Street'})<-[r:ACTED_IN]-(actor)
-RETURN r.role
+MATCH (:Person {name: 'Charlie Sheen'})-[:ACTED_IN]->(movie:Movie)<-[:DIRECTED]-(director:Person)
+RETURN movie.title AS movieTitle, director.name AS director
 ----
 
-Returns the `ACTED_IN` roles for the movie `Wall Street`.
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| movieTitle | director
+| "Wall Street" | "Oliver Stone"
+2+|Rows: 1
+|===
+
+[[where-predicates]]
+== MATCH with WHERE predicates
+
+The `MATCH` clause is often paired with a `WHERE` sub-clause, which adds predicates to refine the patterns, making them more specific.
+These predicates are part of the pattern itself, not just filters applied after matching.
+Thus, the `WHERE` clause should always be placed with its corresponding `MATCH` clause.
```

Comment:
```suggestion
Thus, always place the `WHERE` clause with its corresponding `MATCH` clause.
```

tried to avoid "should" and to shorten it with imperative